app/views/admin/instances/index.html.haml

```diff
@@ -1,11 +1,14 @@
 - content_for :page_title do
   = t('admin.instances.title')
 
+- content_for :header_tags do
+  = javascript_pack_tag 'admin', async: true, crossorigin: 'anonymous'
+
 - content_for :heading_actions do
   - if whitelist_mode?
-    = link_to t('admin.domain_allows.add_new'), new_admin_domain_allow_path, class: 'button'
+    = link_to t('admin.domain_allows.add_new'), new_admin_domain_allow_path, class: 'button', id: 'add-instance-button'
```

Comment:
Why not just set `_domain` to `params[:by_domain]` here, avoiding JS entirely?

Reply:
Maybe I haven't been clear. I usually type the domain then click the button without submitting the form. `params[:by_domain]` would be empty in this case since I haven't submitted the form.